curriculum/challenges/english/25-front-end-development/workshop-blog-page/669fdfc9a5e3da42d2376609.md

```diff
@@ -0,0 +1,104 @@
+---
+id: 669fdfc9a5e3da42d2376609
+title: Step 13
+challengeType: 0
+dashedName: step-13
+---
+
+# --description--
+
+Now that you have added the about section, try clicking on the About link to see the page jump down to that section.
+
+The next section in the blog page will be a list of posts talking about Mr.Whiskers.
```

Comment:
```suggestion
The next section in the blog page will be a list of posts talking about Mr. Whiskers.
```

Reply:
@zairahira 

According to cambridge and miraim dictionary it is double-check
https://dictionary.cambridge.org/us/dictionary/english/double-check

Reply:
Not compulsory though, but we can change `double check` to `double-check` through out the workshop.